site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,245 @@
++++
+date = "2018-05-25T13:00:00+05:30"
+title = "Citra Progress Report 2018 Q1"
+tags = [ "progress-report" ]
+author = "CaptV0rt3x"
+forum = 0000
++++
+
+It's been a while folks. Our content writer [***Anodium***](https://github.com/anodium)
+has had to take a break to deal with the aftermath of hurricane Maria. So, we
+haven't had anyone to write up a progress report till now.
+
+Coming to Citra, since the last report, we have had many new features.
+Some notable ones include multiplayer, hardware shader improvements (post GLvtx),
+a logging system rewrite and the highly coveted camera support. Our continuous integration (CI)
+systems were optimized as well. Apart from these, we have many more minor features,
+improvements and bug fixes.
+
+***Now let’s see these in detail:***
+
+## Citra-qt : Multiplayer support by [jroweboy](https://github.com/jroweboy), [B3N30](https://github.com/B3N30), [Subv](https://github.com/subv) and [JayFoxRox](https://github.com/jayfoxrox)
+
+Citra has been able to run many games at playable speeds for a while now. But it
+was always lacking something, which the 3DS had. The ability to play with friends.
+In order to make a reality our developers [jroweboy](https://github.com/jroweboy),
+[B3N30](https://github.com/B3N30), [Subv](https://github.com/subv) and
+[JayFoxRox](https://github.com/jayfoxrox) worked tirelessly for months to reverse
+engineer the workings of 3DS local wireless system. Thanks to their efforts, Citra
+now emulates _local wireless multiplayer_ over the internet. This allows you to
+play your favorite games with your virtual friends across the world.
+
+#### **Citra multiplayer features**:
+* We now have multiple servers hosted by our very own [Flame Sage](https://github.com/chris062689),
+and many other community members at various locations around the globe, so that
+players can meet others online.
+* Rooms which can hold up to 16 players can be created and even be secured with a
+password.
+* You can now also set your preferred game.
+* Since we now have many number of servers, you can also filter the servers to
+suit your needs.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/Server_list.png"
+    title="Sooo many servers to choose from!" >}}
+
+Since its initial release in the Canary, developers have listened to
+the community and tried very hard to fix bugs affecting many games which weren't
+compatible at that time. And they have also added other features to make the multiplayer
+experience richer. We are proud to announce that multiplayer currently works in
```

Comment:
@CodingKoopa  plz elaborate

Reply:
I was saying that the "We are proud to announce that multiplayer currently works in" sentence would no longer fit in this header after addressing my reviews. Since you moved that to the "Compatible Titles" header, you can disregard this comment.